components/backup_text/Cargo.toml

```
@@ -16,13 +16,17 @@ protobuf-codec = [
 [dependencies]
 engine_rocks = { path = "../engine_rocks", default-features = false }
 engine_traits = { path = "../engine_traits", default-features = false }
+keys = { path = "../keys", default-features = false }
 tikv_util = { path = "../tikv_util", default-features = false }
 tipb = { git = "https://github.com/pingcap/tipb.git", default-features = false }
 tidb_query_datatype = { path = "../tidb_query_datatype", default-features = false }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.66"
```

```suggestion
serde_json = "1.0"
```

`serde_json` is used by several components. Please don't pin the version to `1.0.66`.